### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by ngv557** [NumEconCopenhagen/projects-2020-hazardine-zidane#3]
The structure of the coding is nice and intuitive, great job on that.
Unfortunately, I can’t run any of your codes, it seems the dataset ‘Eksport dataproject’ can’t be executed.
It would be nice to have some comments on all the figures, what are they showing, what stands out. Maybe a comment about the visualization of the financial crisis on the graphs.
I think there are a lot of the tables showing the same thing. Maybe some of them could be left out.
Your README file is incomplete.
In conclusion, great job on coding and structure, need a little finishing touches on the visualization and comments.
Kind regards
Bettina
